sections/sections-app/src/webapp/options.jsp

```diff
@@ -23,6 +28,10 @@
         	</h:panelGrid>
         </t:div>
 
+		<div id="dialog-confirm" title="<h:outputText value="#{msgs.options_manually_manage}"/>" style="display:none">
+		  <p><span class="ui-icon ui-icon-alert" style="float:left; margin:12px 12px 20px 0;"></span><h:outputText value="#{msgs.options_confirmInternal}"/></p>
```

Comment:
can you move this style into a css class?

Reply:
Updated